src/utils_lua.c

```diff
@@ -274,38 +235,200 @@ int luaC_pushvalue(lua_State *L, value_t v, int ds_type) /* {{{ */
   return 0;
 } /* }}} int luaC_pushvalue */
 
-int luaC_pushvaluelist(lua_State *L, const data_set_t *ds,
-                       const value_list_t *vl) /* {{{ */
+int luaC_pushlabelset(lua_State *L, label_set_t label) /* {{{ */
```

Comment:
Not in the header file ⇒ make `static`

Dito for all other new functions except `luaC_pushmetricfamily`.

Reply:
Fixed it. Thanks!